frameworks/Java/redkale/conf/persistence.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence version="2.0" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
+    
+    <persistence-unit name="" transaction-type="RESOURCE_LOCAL">
+        <shared-cache-mode>ALL</shared-cache-mode>
+        <properties>
+            <property name="javax.persistence.jdbc.url" value="jdbc:postgresql://TFB-database:5432/hello_world"/>
```

Comment:
Oh we changed this to `tfb-database` (lowercase).  I think that's why the travis logs are complaining about here: https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/365418054

You can push new commits to your branch and travis will automatically build them, by the way.  You don't need to open a new PR.